### PR TITLE
Persist isEmulator for reload of VSCode, remove graph from quickpick

### PR DESCRIPTION
**Pull request by PrashanthCorp**
Ensures that on reloading VSCode, the attached emulator does not forget its emulator-ness.

### CODE REVIEW

File: src/tree/AttachedAccountsTreeItem.ts [hunk 1]
```diff
@@ -111,7 +111,7 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
 
     public async attachEmulator(): Promise<void> {
         let connectionString: string;
-        const defaultExperience = <Experience>await vscode.window.showQuickPick(['MongoDB', 'DocumentDB', 'Graph'], { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
```
Comment (PrashanthCorp) [inline src/tree/AttachedAccountsTreeItem.ts]: Removing Graph from quick pick

File: src/tree/AttachedAccountsTreeItem.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
                 }
                 const label = `${defaultExperience} Emulator`
                 let treeItem: IAzureTreeItem = await this.createTreeItem(connectionString, defaultExperience, label);
-                if (treeItem instanceof DocDBAccountTreeItem || treeItem instanceof GraphAccountTreeItem || treeItem instanceof TableAccountTreeItem) {
+                if (treeItem instanceof DocDBAccountTreeItem || treeItem instanceof GraphAccountTreeItem || treeItem instanceof TableAccountTreeItem || treeItem instanceof MongoAccountTreeItem) {
```
Comment (PrashanthCorp) [inline src/tree/AttachedAccountsTreeItem.ts]: Left the checks in..

File: src/tree/AttachedAccountsTreeItem.ts [hunk 3]
```diff
@@ -246,18 +249,23 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
...
+                isEmulator = node.isEmulator;
             } else if (node instanceof GraphAccountTreeItem) {
                 experience = Experience.Graph;
+                isEmulator = node.isEmulator;
             } else if (node instanceof TableAccountTreeItem) {
                 experience = Experience.Table;
+                isEmulator = node.isEmulator;
             } else if (node instanceof DocDBAccountTreeItem) {
                 experience = Experience.DocumentDB;
+                isEmulator = node.isEmulator;
```
Comment (PrashanthCorp) [inline src/tree/AttachedAccountsTreeItem.ts]: Should I place all the `isEmulator = node.isEmulator` lines within one large check for all AccountTreeItems?

File: src/docdb/tree/DocDBAccountTreeItemBase.ts [hunk 4]
```diff
@@ -18,8 +18,8 @@ export abstract class DocDBAccountTreeItemBase extends DocDBTreeItemBase<Databas
     public readonly label: string;
     public readonly childTypeLabel: string = "Database";
 
-    constructor(id: string, label: string, documentEndpoint: string, masterKey: string) {
-        super(documentEndpoint, masterKey);
+    constructor(id: string, label: string, documentEndpoint: string, masterKey: string, isEmulator?: boolean) {
```
Comment (StephenWeatherford) [inline src/docdb/tree/DocDBAccountTreeItemBase.ts]: Why is isEmulator optional?

File: src/tree/AttachedAccountsTreeItem.ts [hunk 5]
```diff
@@ -111,7 +111,7 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
 
     public async attachEmulator(): Promise<void> {
         let connectionString: string;
-        const defaultExperience = <Experience>await vscode.window.showQuickPick(['MongoDB', 'DocumentDB', 'Graph'], { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
+        const defaultExperience = <Experience>await vscode.window.showQuickPick(['MongoDB', 'DocumentDB'], { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
```
Comment (StephenWeatherford) [inline src/tree/AttachedAccountsTreeItem.ts]: Please add a known issue to CHANGELOG.md and log an issue to support later.

File: src/docdb/tree/DocDBAccountTreeItemBase.ts (hunk 4 above)
Comment (PrashanthCorp) [reply src/docdb/tree/DocDBAccountTreeItemBase.ts]: The `CosmosDBAccountProvider` class uses this constructor, and currently, I see no use of adding an `isEmulator` field there.

**Pull request merged**